### PR TITLE
Add `Move` implementation

### PR DESCRIPTION
**Pull request by blazej-smorawski**

### CODE REVIEW

File: lynx/common/actions/move.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
 from dataclasses import dataclass
```
Comment (filip-szweda) [inline lynx/common/actions/move.py:1]: > Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

https://peps.python.org/pep-0008/#imports
Just do that in the files that you have changed. I will do the rest later.

File: lynx/common/scene.py [hunk 2]
```diff
@@ -11,13 +11,27 @@
 class Scene(Serializable):
     entities: List[Entity] = field(default_factory=list)
     _objects_map: Dict[Vector, Object] = field(default_factory=dict)
+    _id_map: Dict[int, Object] = field(default_factory=dict)
```
Comment (filip-szweda) [inline lynx/common/scene.py]: Please rename to `_position_entities_map` and `_id_entities_map`.

File: lynx/common/scene.py [hunk 3]
```diff
@@ -11,13 +11,27 @@
...
                 self._objects_map[entity.position] = entity
+                self._id_map[entity.id] = entity
 
     def add_entity(self, entity: Entity) -> NoReturn:
         self.entities.append(entity)
         if type(entity) is Object:
             self._objects_map[entity.position] = entity
+            self._id_map[entity.id] = entity
+
+    def get_object_by_id(self, id: int) -> (Object | None):
```
Comment (filip-szweda) [inline lynx/common/scene.py]: You can use `Union` return type from `typing` like e.g. `-> Union[Object, None]`.

**Review by filip-szweda — COMMENTED** [review]
Nice and clean. GJ.

File: lynx/common/scene.py (hunk 3 above)
Comment (blazej-smorawski) [reply lynx/common/scene.py]: Is it wrong? I think I've seen people use sth like that.

File: lynx/common/scene.py (hunk 2 above)
Comment (blazej-smorawski) [reply lynx/common/scene.py]: These are quite long, maybe `_position_map` and `_id_map`?

Comment (filip-szweda) [reply lynx/common/scene.py]: I wouldn't want those, because they sound as if we only kept positions and ids in these dicts. And that's not true, because entities are the focus.
Long names are not a problem at all, as long as they're descriptive.

File: lynx/common/scene.py (hunk 3 above)
Comment (filip-szweda) [reply lynx/common/scene.py]: ~~Nah. It's just that we already import `typing` in most of our files, so we may as well use it for consistency's sake. It's more Pythonic.
If you feel like it'a a _whatever_ thing then I'm not going to chase you to change it xD~~
`-> (Object | None)` seems to be broken.

File: lynx/common/scene.py [hunk 4]
```diff
@@ -11,13 +11,27 @@
...
+        return self._id_map.get(id)
+    
+    def get_object_by_position(self, position: Vector) -> (Object | None):
+        return self._object_map.get(position)
+    
+    def move_object(self, object: Object, vector: Vector) -> NoReturn:
+        self._objects_map[object.position] = None
```
Comment (EasyJB) [inline lynx/common/scene.py]: Do we need this None assignment?

File: lynx/common/scene.py (hunk 3 above)
Comment (filip-szweda) [reply lynx/common/scene.py:28]: `-> (Object | None)` seems to be broken.

**Review by EasyJB — COMMENTED** [review]
Looking good

File: lynx/common/scene.py (hunk 4 above)
Comment (alpinus4) [reply lynx/common/scene.py]: yeah, we gotta clear the old position

Comment (EasyJB) [reply lynx/common/scene.py]: I se, nvm i forget that we need to clear the previous field

File: setup.py
Comment (alpinus4) [inline setup.py:1]: Find packages got nothing to do with Move action
Should be in separate PR

Comment (blazej-smorawski) [reply setup.py:1]: I think that separate commit is enough, who cares in the end what PR brought that in

File: lynx/common/scene.py (hunk 2 above)
Comment (blazej-smorawski) [reply lynx/common/scene.py]: Not a huge fan of long names, but I guess u're right :). What do you think bout _object_position_map + _object_id_map? I think I rather having it in a format `value_key_map`

File: lynx/common/actions/move.py (hunk 1 above)
Comment (blazej-smorawski) [reply lynx/common/actions/move.py:1]: Done

File: lynx/common/scene.py (hunk 3 above)
Comment (blazej-smorawski) [reply lynx/common/scene.py]: Done

File: lynx/common/scene.py (hunk 2 above)
Comment (filip-szweda) [reply lynx/common/scene.py]: That's good 👍
